docs/source/reference/io.rst

```diff
@@ -34,6 +34,7 @@ String formatting
 .. autosummary::
    :toctree: generated/
 
+   format_float_positional
```

Comment:
Please move this to the line after `array_str` to be consistent with NumPy's reference.
https://numpy.org/doc/stable/reference/routines.io.html

Reply:
Noted.